tests/wdio/pageobjects/configuration.page.js

```diff
@@ -0,0 +1,557 @@
+const Page = require('../vaah-webdriverio/Page');
+const Sl = require('../vaah-webdriverio/Selector');
+const asserts = require('../vaah-webdriverio/Assert')
+
+class ConfigurationPage extends Page{
+    constructor() {
+        super();
+        this.params.page.id = "CG";
+        this.params.page.name = "Configuration";
+        this.params.page.path = "/backend#/setup/install/configuration";
+        this.params.page.url = this.base_url+this.params.page.path;
+    }
+
+    async open()
+    {
+        await browser.maximizeWindow();
+        await asserts.pause();
+        await super.open(this.params.page.url);
+    }
+
+    async clear(element){
```

Comment:
@we-abhijeet-g001 This method should not be part of this classes, because it can be used other classes as well. create a new claas `tests/wdio/vaah-webdriverio` as `Helpers.js` and move the method and use that

Reply:
And rename it from `clear` to `VueClear`